packages/adapters/wagmi/src/client.ts

```diff
@@ -701,6 +705,7 @@ export class WagmiAdapter implements ChainAdapter {
 
             this.appKit?.setPreferredAccountType(preferredAccountType, chainNamespace)
             this.appKit?.setCaipAddress(caipAddress, chainNamespace)
+            this.appKit?.setStatus('connected', chainNamespace)
```

Comment:
Maybe not check for connected and set the state directly?

Reply:
Done.